sdk/src/test/scala/com/horizen/chain/SidechainBlockInfoTest.scala

```diff
@@ -25,6 +26,14 @@ class SidechainBlockInfoTest extends JUnitSuite with SidechainBlockInfoFixture {
   val semanticValidity: ModifierSemanticValidity = ModifierSemanticValidity.Valid
   val mainchainHeaderHashes: Seq[MainchainHeaderHash] = Seq("0269861FB647BA5730425C79AC164F8A0E4003CF30990628D52CEE50DFEC9213", "E78283E4B2A92784F252327374D6D587D0A4067373AABB537485812671645B70",
```

Comment:
can be removed, as we have `mainchainHeadersBaseInfo` now.
Also use the `mainchainHeadersBaseInfo` for the checks below.